rosidl_typesupport_introspection_cpp/cmake/rosidl_typesupport_introspection_cpp_generate_interfaces.cmake

```diff
@@ -62,6 +62,8 @@ add_library(${rosidl_generate_interfaces_TARGET}${_target_suffix} SHARED ${_gene
 if(WIN32)
   target_compile_definitions(${rosidl_generate_interfaces_TARGET}${_target_suffix}
     PRIVATE "ROSIDL_TSI_CPP_BUILDING_DLL")
+  target_compile_definitions(${rosidl_generate_interfaces_TARGET}${_target_suffix}
+    PRIVATE "ROSIDL_BUILDING_DLL")
```

Comment:
I think I actually meant to remove the flag above, along with the whole visibility header in the `rosidl_typesupport_introspection_cpp` package. I'll update it one way or another.